pkg/bulkAction/BulkUpdateService.go

```diff
@@ -1122,13 +1149,13 @@ func (impl BulkUpdateServiceImpl) BulkUnHibernate(request *BulkApplicationForEnv
 		impl.logger.Errorw("error in fetching pipelines", "envId", request.EnvId, "err", err)
 		return nil, err
 	}
-	response := make(map[string]map[string]bool)
+	response := make(map[string]map[string]any)
```

Comment:
we don't use generics much, please add comment on why this is being used here

Reply:
because the above function needs to return two types of values, strings and booleans, using generic type for map will allow us to have both types in a single map